File: Programmers/Level1/짝지어 제거하기/짝지어 제거하기.java
import java.util.*;
class Solution
{
    public int solution(String s){
        int answer=0;
        Stack<Integer> stack = new Stack<>();
        for(int i=0; i<s.length(); i++){
            if(!stack.empty() && stack.peek()==(int)s.charAt(i)){
                stack.pop();
            }else{
                stack.push((int)s.charAt(i));
            }
            
        }

        if(stack.empty()){
            answer=1;
        }
        return answer;
    }
}

File: Programmers/Level2/짝지어 제거하기/짝지어 제거하기.py
def solution(s): 
    arry = []    
    for i in s:
        
        if len(arry) == 0: 
            arry.append(i)
        elif arry[-1] == i: 
            arry.pop()
        else: 
            arry.append(i)
            
    if len(arry) == 0: 
        return 1
    else: 
        return 0
